### PR TITLE
readme sleuth comparison

### PR DESCRIPTION
**Pull request by pavolloffay**
@objectiser could you please review?

@tedsuo here is the description how it differs from sleuth. The next step is to create an umbrella issue in /meta.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,6 +5,16 @@
 This library provides instrumentation for Spring  Web applications (Boot and MVC). It creates tracing data for 
 server requests and also client requests (`RestTemplate` and `AsyncRestTemplate`).
 
+## Comparison to `spring-cloud-sleuth`
+This project provides instrumentation only for `spring-web` package. In other words it traces only
+HTTP requests made to Spring Boot/WEB app and outgoing requests using Spring RestTemplate. However it allows
+you to use OpenTracing API directly in your code and combine different OpenTracing instrumentations together
+very easy (e.g. [OpenFeign](https://github.com/OpenFeign/feign-opentracing)).
```
Comment (objectiser) [inline README.md]: "very easy" -> "easily" ?

File: README.md [hunk 2]
```diff
@@ -5,6 +5,16 @@
...
+
+Whereas [spring-cloud-sleuth](https://github.com/spring-cloud/spring-cloud-sleuth) combines
+instrumentations for different frameworks together and you are not allowed to use OpenTracing API or wire
```
Comment (objectiser) [inline README.md]: Full stop after "together". And then possibly "It is not currently possible to use the OpenTracing API (and therefore switch between tracing backends), or wire in different instrumentations that are not suported by sleuth ..." - possible slight rewording.

Comment (pavolloffay) [inline README.md]: looks good :+1:

**Comment by pavolloffay** [comment]
@objectiser thanks!

**Comment by tedsuo** [comment]
great, thanks!

**Pull request merged**